LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -188,7 +188,16 @@ public bool HasPassword(User user)
         /// <inheritdoc />
         public Task ChangePassword(User user, string newPassword)
         {
-            throw new NotImplementedException();
+            if (!LdapPlugin.Instance.Configuration.AllowPassChange)
+            {
+                throw new NotImplementedException();
+            }
+
+            var passAttr = LdapPlugin.Instance.Configuration.LdapPasswordAttribute;
+            var ldapUser = LocateLdapUser(user.Username) ?? throw new LdapException("No users found in LDAP Query");
```

Comment:
I would prefer a null check like L66

Reply:
Would it make more sense to move the null check into the function?

Reply:
[83689e3](https://github.com/jellyfin/jellyfin-plugin-ldapauth/pull/115/commits/83689e36b2d2b40391dc678df462c5542d9b8801)